src/Generating/Trees.cpp

```diff
@@ -29,6 +29,38 @@ static const sCoords Corners[] =
 	{1, 1},
 } ;
 
+// Array with possible directions for a branch to go to.
```

Comment:
No longer an array; change to doxy-comment and document the individual parameters and return value.